### PR TITLE
Adapt code for fixed protocol interface

### PR DESCRIPTION
**Pull request by patnorris**
Deleting Bridges in Bebb Protocol needed fixing. This PR adds the updated interface.

To test:
Create Neighbors for a Space and make sure you can delete them (no errors during deletion and when reloaded, Neighbor is gone).
The code for OIM and Bebb Protocol is deployed on the mainnet.

### CODE REVIEW

File: src/integrations/BebbProtocol/newwave.did [hunk 1]
```diff
@@ -32,6 +32,20 @@ type Request =
    method: text;
    url: text;
  };
+type NewWaveError =
```
Comment (tiusty) [inline src/integrations/BebbProtocol/newwave.did:35]: What is new wave again? Is that the name of a connection?

File: src/integrations/BebbProtocol/newwave.did [hunk 2]
```diff
@@ -117,16 +136,11 @@ type BridgeEntity =
    toEntityId: text;
  };
 service : {
-  createBridge: (BridgeEntityInitiationObject) -> (opt BridgeEntity);
```
Comment (tiusty) [inline src/integrations/BebbProtocol/newwave.did:120]: Why were there duplicate functions before? Was there multiple functions defined in the Bebb protocol before?

**Review by tiusty — APPROVED** [review]
Looks good, tested and was able to delete neighbors

File: src/integrations/BebbProtocol/newwave.did (hunk 1 above)
Comment (patnorris) [reply src/integrations/BebbProtocol/newwave.did:35]: :) "NewWave" was the working title before "Bebb Protocol", so it's the same, though confusing. For future versions, we can update all code and references with the new name

**Pull request merged**